go.sum

```diff
@@ -120,16 +120,18 @@ github.com/ionos-cloud/sdk-go-bundle/products/cert/v2 v2.2.0 h1:UAsgMawKobayMkf7
 github.com/ionos-cloud/sdk-go-bundle/products/cert/v2 v2.2.0/go.mod h1:7ONrlRb2n7qjLR5O7oCQzOJAiRhj/D4ywvpJUPQr1q0=
 github.com/ionos-cloud/sdk-go-bundle/products/containerregistry/v2 v2.1.0 h1:Jukq9lOxyd3pgG2HE8OVu+CbMIV8PGXIqFcnN66pDJU=
 github.com/ionos-cloud/sdk-go-bundle/products/containerregistry/v2 v2.1.0/go.mod h1:MqxTZ40QJv2yq2qlNAzIABRwCKV4oDf7tRSbtwQIk0g=
+github.com/ionos-cloud/sdk-go-bundle/products/dbaas/mongo/v2 v2.1.0 h1:Ial6fztU2ilhX0CXjfqt4i9d4R1WE1tgQEuzRNApB00=
+github.com/ionos-cloud/sdk-go-bundle/products/dbaas/mongo/v2 v2.1.0/go.mod h1:cI558LGis0dz/l1CiCIpGLI9gmcBqZenCeg2GYPpZrM=
 github.com/ionos-cloud/sdk-go-bundle/products/dns/v2 v2.2.0 h1:LsRmU3eb6MeI+t3OP3kEVvP0TSp0UzPRmw1GWLRbu/0=
 github.com/ionos-cloud/sdk-go-bundle/products/dns/v2 v2.2.0/go.mod h1:S8SBfF2bJkW902JKSQ/O6zn3sXPgeLvaa8hUG/oycNg=
 github.com/ionos-cloud/sdk-go-bundle/products/logging/v2 v2.1.1 h1:GXxdNKl7haQnHO1QewZeXazur5+LDCpHqvP+nNObGaA=
 github.com/ionos-cloud/sdk-go-bundle/products/logging/v2 v2.1.1/go.mod h1:N0ifRq3ENqGg6qht1WPrVgSkcavPS5SRWlNfkHUZbfQ=
 github.com/ionos-cloud/sdk-go-bundle/products/monitoring/v2 v2.0.0 h1:gVh+jd0o8ROtJLWqUB6/VU8vSLdj/kRYAz/qbMO7z7I=
 github.com/ionos-cloud/sdk-go-bundle/products/monitoring/v2 v2.0.0/go.mod h1:PRdc/x/r7tMEV7rh2uZYJ8ososjfIF/cXiGX/D6FdTs=
-github.com/ionos-cloud/sdk-go-bundle/products/objectstoragemanagement/v2 v2.0.0 h1:6xP8S4gGu+n8xpvyvVuWjWljeuPduCA+mvpb2rgqDt0=
-github.com/ionos-cloud/sdk-go-bundle/products/objectstoragemanagement/v2 v2.0.0/go.mod h1:Zzv2k/EXEYp+nwGk2s86jkvUJLa4ebWJoOdGfYcb8mQ=
```

Comment:
@rmocanu-ionos again, let's double check if `go.mod` is correct since here I see some changes related to `objectstoragemanagement` (I didn't expect to see these).